ledfx/effects/hierarchy.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import timeit
+
+import numpy as np
+import voluptuous as vol
+
+from ledfx.color import parse_color, validate_color
+from ledfx.effects.audio import AudioReactiveEffect
+from ledfx.effects.gradient import GradientEffect
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Filter(AudioReactiveEffect, GradientEffect):
+    NAME = "Hierarchy"
+    CATEGORY = "Simple"
+    HIDDEN_KEYS = [
+        "background_color",
+        "background_brightness",
+        "blur",
+        "mirror",
+        "flip",
+        "gradient_roll",
+        "gradient",
+        "frequency_range",
+        "brightness",
+    ]
+    ADVANCED_KEYS = []
+
+    CONFIG_SCHEMA = vol.Schema(
+        {
+            vol.Optional(
+                "color_lows",
+                description="Color of low, bassy sounds",
+                default="#FF0000",
+            ): validate_color,
+            vol.Optional(
+                "color_mids",
+                description="Color of midrange sounds",
+                default="#00FF00",
+            ): validate_color,
+            vol.Optional(
+                "color_high",
+                description="Color of high sounds",
+                default="#0000FF",
+            ): validate_color,
+            vol.Optional(
+                "brightness_boost",
+                description="Multiplier for the Brightness",
+                default=1.0,
+            ): vol.All(vol.Coerce(float), vol.Range(min=0.0, max=4.0)),
+            vol.Optional(
+                "switch_threshold_lows",
```

Comment:
be cautious of long field names, see how they scale for a narrow browser view, or from a phone